pages/common/pdftk.md

```diff
@@ -0,0 +1,23 @@
+# pdftk
+
+> PDF toolkit.
+
+- Extract pages from a PDF file and save them as another one:
+
+`pdftk {{input.pdf}} cat {{1-3 5 6-10}} output {{output.pdf}}`
+
+- Merge (concatenate) a list of PDF files and save the result as another one:
+
+`pdftk {{file1.pdf}} {{file2.pdf}} ... cat output {{output.pdf}}`
+
+- Separate a PDF file into individual ones, with a given output pattern:
```

Comment:
I liked that the previous version of this description mentioned that the splitting was done per page. How about something like "Split each page of a PDF file into a separate file, with a given output filename pattern"?